core/src/avm2/globals/flash/text/engine/GroupElement.as

```diff
@@ -0,0 +1,85 @@
+package flash.text.engine {
+    import __ruffle__.stub_method;
+
+    import flash.events.EventDispatcher;
+
+    public final class GroupElement extends ContentElement {
+        internal var _elements = null;
+
+        public function GroupElement(elements:Vector.<ContentElement> = null, elementFormat:ElementFormat = null, eventMirror:EventDispatcher = null, textRotation:String = "rotate0") {
+            super(elementFormat, eventMirror, textRotation);
+            this.setElements(elements);
+        }
+
+        public function get elementCount():int {
+            return this._elements.length;
+        }
+
+        public function getElementAt(index:int):ContentElement {
+            if (index < 0 || index >= this._elements.length) {
+                throw new RangeError("Error #2006: The supplied index is out of bounds.", 2006);
+            }
+            return this._elements[index];
+        }
+
+        public function getElementIndex(element:ContentElement):int {
+            return this._elements.indexOf(element);
+         }
+
+        public function setElements(elements:Vector.<ContentElement>):void {
+            if (elements == null) {
+                this._elements = new Vector.<ContentElement>();
+            } else {
+                this._elements = elements.AS3::concat();
+            }
+        }
+
+        public function replaceElements(beginIndex:int, endIndex:int, newElements:Vector.<ContentElement>):Vector.<ContentElement> {
+            // This some sort of special case that doesn't throw.
+            if (beginIndex == endIndex && newElements == null) {
+                return null;
+            }
+            if (beginIndex < 0 || beginIndex > this._elements.length ||
```

Comment:
Should these `>`s be replaced with `>=`s?

Reply:
I don't think so. AS seems to allow adding new elements at the end. Also tested by 
> /// replaceElements(3, 3, replacement)